### ISSUE TITLE
Show correct PromotedStories if progress is available but not for last week.

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#1562]
Currently `PromotedStories` work in this way:
1. If learner is opening the application for the first time, in this case because there is no progress we show recommended stories (basically first story of each topic.)
2. If learner has made some progress in last 7 days, then we add those stories as part of `recentStories` in `OngoingStoryList`.
3. If learner has made some progress but not in last 7 days, then we add those stories as part of `oldStories` in `OngoingStoryList` but it is not visible in Home screen.

So, basically the (3.) point is an issue, and we should fix this to show those stories in home screen.
